haproxy_exporter.go

```diff
@@ -358,13 +359,31 @@ func main() {
 		haProxyScrapeUri          = flag.String("haproxy.scrape_uri", "http://localhost/;csv", "URI on which to scrape HAProxy.")
 		haProxyServerMetricFields = flag.String("haproxy.server_metric_fields", "", "If specified, only export the given csv fields. Comma-seperated list of numbers. See http://cbonte.github.io/haproxy-dconv/configuration-1.5.html#9.1")
 		haProxyTimeout            = flag.Duration("haproxy.timeout", 5*time.Second, "Timeout for trying to get stats from HAProxy.")
-		_                         = flag.Duration("haproxy.scrape_interval", 0, "DEPRECATED. Not used anymore.")
+		haProxyPidFile            = flag.String("haproxy.pid_file", "", "Path to haproxy's pid file.")
```

Comment:
Btw., the new flag-naming standard should be dashes now instead of underscores (as in https://github.com/prometheus/prometheus/blob/master/main.go#L45-L66). Maybe a good opportunity to update all of them before launch?